FreeRTOS/Test/CMock/smp/single_priority_timeslice/single_priority_timeslice_utest.c

```diff
@@ -911,20 +852,21 @@ void test_task_suspend_running_task( void )
  * Priority – 2               Priority – 2
  * State - Running (Core N)   State - Ready
  * 
- * Call xTaskIncrementTick() for each configured CPU core. The kernel will consider
- * CPU 0 the core calling the API and therefore will not rotate tasks for that CPU.
+ * Call xTaskIncrementTick() for each configured CPU core.
  * 
  * Task (TN + 1) when configNUMBER_OF_CORES = 4
  * Tick    Core
- * 1       1
- * 2       2
- * 3       3
- * 4      -1
+ * 1       0
+ * 2       1
+ * 3       2
+ * 4       3
  * 
- * Suspend the last created task
+ * Delay the task running on core 0, which is task 1.
  * 
  * Call xTaskIncrementTick() for each configured CPU core. The tasks will not
  * change state.
+ *
+ * After delay, verify task 1 can be scheduled on each core.
```

Comment:
Minor nitpick, can you update this comment and the ones below to say 'blocked' instead of 'delayed'. This would be consistent with the test case name, and the fact that even though we are calling vTaskDelay() we are putting the task into the blocked state (as verified in the test).